boyhood.png

```diff
@@ -0,0 +1 @@
+movie30-2025/boyhood.png
```

Comment:
symlink -- actually not sure if github will handle this properly. we can hard-copy it if need be.